charts/stolon/templates/components/keeper/statefulset.yaml

```diff
@@ -112,6 +112,44 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /stolon/data
+      {{- if .Values.exporter.enabled }}
+      - name: exporter
+        image: {{ .Values.exporter.image.repository }}:{{ .Values.exporter.image.tag }}
+        imagePullPolicy: {{ .Values.exporter.image.pullPolicy }}
+        args:
+        {{- if .Values.exporter.disableDefaultMetrics }}
+        - "--disable-default-metrics"
+        {{- end }}
+        {{- if .Values.exporter.disableSettingsMetrics }}
+        - "--disable-settings-metrics"
+        {{- end }}
```

Comment:
For unification purposes I think we should use appropriate env variables instead of command line arguments

Reply:
resolved